### DISCUSSION TITLE
Draft new release for existing formula

### DISCUSSION THREAD

**Discussion by jcamiel** [Homebrew/discussions#3879] [Tap maintenance and Homebrew development] [answered]
Hi,

I'm maintainer of Hurl (https://github.com/Orange-OpenSource/hurl) a CLI HTTP client.
Hurl is available on Brew with this formula: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/hurl.rb

We've released a draft GitHub new release of Hurl (1.8.0). For the moment, it's still a pre-release (we're doing tests on it). I was worried to see a PR on this new release (1.8.0) https://github.com/Homebrew/homebrew-core/pull/114698 initiated by @chenrui333 (big thanks to him by the way). It's tagged `pre-release` but I am worried the formula being updated while our release is still in draft.

What's the good process for a maintainer like me to signal that there is a new official release? Actually, we prepare our release with a GitHub draft release, and a tag on a git branch (for instance `1.8.0` on branch `release-1.8.0`). One we think our release is good, we remove the "Draft" status from the GitHub release, merge on master and move the new tag on master (`1.8.0` on branch `master`).

What's the best practices for my project? I'm wondering also what's triggering the process of bumping a Brew formula?

Thanks,

 Jc

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Updating-Software-in-Homebrew documents the process of updating software in Homebrew. As you noticed there is nothing stopping people from making a pull request for a draft release, but we do have automation to stop such release pull requests from being merged.

Additionally you could add a `github_latest` setting in a `livecheck` block. That should stop suggesting these releases for pull requests
